semver/range_test.ts

```diff
@@ -71,7 +70,6 @@ Deno.test("range", async (t) => {
     ["1.2.3 >=1.2.1", "1.2.3"],
     [">=1.2.3 >=1.2.1", "1.2.3"],
     [">=1.2.1 >=1.2.3", "1.2.3"],
-    ["<=1.2.3", "1.2.3-beta"],
```

Comment:
This case is also incompatible `npm:semver`. The below prints false, but expected true here.

```js
import * as semver from "npm:semver";
const range = new semver.Range("<=1.2.3");
console.log(range.test("1.2.3-beta"));
```

Reply:
Ditto. Is this `npm:semver` present somewhere?

Reply:
Added this case back to `testRange() returns false when the version is not in the range` with fixed expectation